sdk/translation/Azure.AI.Translation.Text/README.md

```diff
@@ -103,10 +103,10 @@ The following section provides several code snippets using the `client` [created
 
 Gets the set of languages currently supported by other operations of the Translator.
 
-```C# Snippet:GetTextTranslationLanguagesAsync
+```C# Snippet:GetTextTranslationLanguages
```

Comment:
Can you please go thru all MD files for this Text Translation project and make sure that we have update the text from referencing asynchronous methods to sync. See example on line 100 in this document:

_Although most of the snippets below make use of asynchronous service calls, keep in mind that the `Azure.AI.Translation.Text` package supports both synchronous and asynchronous APIs._

Reply:
Corrected README.md and checked all the sample.md files for references to async code.